spec/lib/id3tag/frames/v2/url_frame_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "spec_helper"
+
+describe ID3Tag::Frames::V2::UrlFrame do
+  let(:id) { "W" }
+  let(:raw_content) { url }
```

Comment:
In the specification it says:
> If the text string is followed by a string termination, all the following information should be ignored and not be displayed

This spec file could be improved with test case for that. Similar functionality has been implemented in `UserTextFrame`.